package.json

```diff
@@ -10,7 +10,7 @@
     "test-cypress": "wait-on http://localhost:8090 && cypress run",
     "pretest-ci": "npm run build",
     "test-ci": "./bin/start-test-server & concurrently --success --raw 'npm test' 'npm run test-cypress'",
-    "build": "gulp build && webpack",
+    "build": "gulp clean && gulp css && webpack",
```

Comment:
Being explicit with these tasks means we can remove some code from the gulpfile.